### PR TITLE
Review guidelines

### PR DESCRIPTION
**Pull request by aburrell**
Add reviewer guidelines into develop.

### CODE REVIEW

File: Email_Forms/acknowledge_receipt_of_review.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Notes
+
+In the following form, repace:
```
Comment (mjonesjr) [inline Email_Forms/acknowledge_receipt_of_review.md:3]: Change repace to replace.

File: Email_Forms/acknowledge_receipt_of_review.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Dear Reviewer,
+
+Thank you for your review of this recommendation/nomination letter, which we have successfully received. If you have any changes to make to your review, please email those changes to equitable.space.letters@gmail.com.
+
+We sincerely appreciate the time and effort you put into this review. Without your careful and thoughtful review of letters such as this, we would not be able to bring forth actionable change on issues of diversity, equity, and inclusion in our community.
+
+Potential statement about demographic information of the reviewer.
+
+Potential state about receiving feedback about the process.
```
Comment (mjonesjr) [inline Email_Forms/acknowledge_receipt_of_review.md:17]: Change state to statement.

File: Email_Forms/receipt_of_submission.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Notes
+
+In the following form, repace:
```
Comment (mjonesjr) [inline Email_Forms/receipt_of_submission.md:3]: Change repace to replace

File: Email_Forms/requesting_review.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Notes
+
+In the following form, repace:
```
Comment (mjonesjr) [inline Email_Forms/requesting_review.md:3]: Change repace to replace.

File: Email_Forms/return_ferpa_to_submitter.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Notes
+
+In the following form, repace:
```
Comment (mjonesjr) [inline Email_Forms/return_ferpa_to_submitter.md:3]: Change repace to replace.

File: Email_Forms/return_ferpa_to_submitter.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+Sender - name of person who submitted a letter for review
+sender@email.com - email of person who submitted a letter for review
+Editor - your name
+Xxxxx (e.g., example) - replace with the type of PII found in the letter and wheere in the letter it was located.  Do not copy the PII in the email.
```
Comment (mjonesjr) [inline Email_Forms/return_ferpa_to_submitter.md:7]: Change wheere to where.

File: Email_Forms/return_ferpa_to_submitter.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+Dear Sender,
+
+Thank you for your recent submission to Equitable Letters in Space Physics. However, we were unable to start the review process until the following personally identifiable information (PII) and or FERPA-protected education record(s) is/are removed or redacted from the letter:
+
+Xxxxx (e.g., Grades in the first paragraph)
+Xxxxx (e.g., Place of birth in second paragraph)
+
+Please review the attached letter and make the necessary modifications.
+
+If you have any further questions regarding this matter, please contact us by replying to the e-mail.
```
Comment (mjonesjr) [inline Email_Forms/return_ferpa_to_submitter.md:20]: Change "the" to "this"

File: Email_Forms/return_review_to_submitter.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+# Notes
+
+In the following form, repace:
```
Comment (mjonesjr) [inline Email_Forms/return_review_to_submitter.md:3]: Change repace to replace.

File: Email_Forms/return_review_to_submitter.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+# Form
+
+Dear Sender,
+
+Thank you for submitting your letter to Equitable Letters in Space Physics. We have received x reviews of you letter, which are included and/or attached. The thoughtful and actionable feedback offered in the reviewer assessment of your letter should be taken into account and will aid you in submitting a more equitable nomination/recommendation letter.
+
+The criteria used to review your letter can be found here (link to our reviewer criteria).
+
+We hope that you find the reviewer comments helpful towards improving the overall quality and reduces the implicit bias associated with the nomination/recommendation letter. If have any additional questions, or need more clarity regarding any of the reviewer comments, please contact us at equitable.space.letters@gmail.com.
```
Comment (mjonesjr) [inline Email_Forms/return_review_to_submitter.md:15]: The second sentence should read "If you ...."

**Review by mjonesjr — CHANGES_REQUESTED** [review]
Hi Angeline,

Just a couple of typos (probably mine) to clean up.

Mack

File: Email_Forms/acknowledge_receipt_of_review.md (hunk 1 above)
Comment (aburrell) [inline Email_Forms/acknowledge_receipt_of_review.md:3]:
```suggestion
In the following form, replace:
```

File: Email_Forms/acknowledge_receipt_of_review.md (hunk 2 above)
Comment (aburrell) [inline Email_Forms/acknowledge_receipt_of_review.md:17]:
```suggestion
Potential statement about receiving feedback about the process.
```

File: Email_Forms/receipt_of_submission.md (hunk 3 above)
Comment (aburrell) [inline Email_Forms/receipt_of_submission.md:3]:
```suggestion
In the following form, replace:
```

File: Email_Forms/requesting_review.md (hunk 4 above)
Comment (aburrell) [inline Email_Forms/requesting_review.md:3]:
```suggestion
In the following form, replace:
```

File: Email_Forms/return_ferpa_to_submitter.md (hunk 5 above)
Comment (aburrell) [inline Email_Forms/return_ferpa_to_submitter.md:3]:
```suggestion
In the following form, replace:
```

File: Email_Forms/return_ferpa_to_submitter.md (hunk 6 above)
Comment (aburrell) [inline Email_Forms/return_ferpa_to_submitter.md:7]:
```suggestion
Xxxxx (e.g., example) - replace with the type of PII found in the letter and where in the letter it was located.  Do not copy the PII in the email.
```

File: Email_Forms/return_ferpa_to_submitter.md (hunk 7 above)
Comment (aburrell) [inline Email_Forms/return_ferpa_to_submitter.md:20]:
```suggestion
If you have any further questions regarding this matter, please contact us by replying to this e-mail.
```

File: Email_Forms/return_review_to_submitter.md (hunk 8 above)
Comment (aburrell) [inline Email_Forms/return_review_to_submitter.md:3]:
```suggestion
In the following form, replace:
```

File: Email_Forms/return_review_to_submitter.md (hunk 9 above)
Comment (aburrell) [inline Email_Forms/return_review_to_submitter.md:15]:
```suggestion
We hope that you find the reviewer comments helpful towards improving the overall quality and reduces the implicit bias associated with the nomination/recommendation letter. If you have any additional questions, or need more clarity regarding any of the reviewer comments, please contact us at equitable.space.letters@gmail.com.
```

**Review by mjonesjr — APPROVED** [review]

**Pull request merged**